### ISSUE TITLE
New command for NVDA

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#2048]
**Reported by jfauchon on 2012-01-17 12:40**
Hello,

May you create a new command which announce how many items there are in list and the number columns, lines in the table and the current position in the table.

Then, may you adapt this command to annouce the current position (current line, column and page) in a document word.

Thanks

**Comment by bhavyashah** [comment]
Don't commands like NVDA+Delete and NVDA+Tab help somewhat in this regard? If not, this may be possibly related to a ticket by Mesar I and @jcsteh recently commented on, whose number I am unable to relocate.

**Comment by Adriani90** [comment]
In lists and tables NVDA already says element position (i.e. in lists it reports 1 of 4, in tables you can hear to the number of rows and columns when entering the table.
Regarding MS Word, there is another issue already in place for that.
I am closing this one.
